project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc

```diff
@@ -517,12 +517,14 @@ function c4m_user_notifications_notify_user_admin_demoted($gid, $uid) {
  *   User id.
  */
 function _c4m_user_notify_user_admin_demoted($gid, $uid) {
```

Comment:
We have 2 functions for this event:
c4m_user_notifications_notify_user_admin_demoted - notify user that was demoted
c4m_user_notifications_notify_admins_admin_demoted - notify GO/GA that admin was demoted